client/src/components/layout/layout.jsx

```diff
@@ -1,17 +1,18 @@
 import React from "react";
 import Header from "./header";
 import BottomNav from "./bottomNav";
-
```

Comment:
These lines are usually there to allow better readability between the imports and the actual code